src/scss/components/_progress.scss

```diff
@@ -63,6 +63,12 @@
                 }
             }
         }
+        &.is-not-rounded {
```

Comment:
I prefer `is-squared` and please don't use !important

Reply:
it cannot set without !important keyword

Reply:
Done!